### ISSUE TITLE
Задание 3 

### ISSUE THREAD

**Issue by djadjka** [AlexZetto/famcs-2018#3]
![default](https://user-images.githubusercontent.com/12067451/36643919-339e6200-1a63-11e8-92a4-8f3fc00f414f.jpg)
https://www.youtube.com/watch?v=oFYyTZwMyAg
https://github.com/ala-n/fpmi-2018/blob/master/pull-request-notes.txt

**Comment by djadjka** [comment]
![image](https://user-images.githubusercontent.com/12067451/36643951-b93c4008-1a63-11e8-8a7b-001363c2b23e.png)
когда я стану богатым и смогу позволить себе iphone x, я не смогу выкладывать фоточки своей кошки в ZGramm

**Comment by AlexZetto** [comment]
А что не так с iphone x?

**Comment by AlexZetto** [comment]
???
![image](https://user-images.githubusercontent.com/31699385/36644486-1a0ea314-1a6c-11e8-8dea-cd2a22a42849.png)

**Comment by djadjka** [comment]
ну, с iphone x 
https://www.youtube.com/watch?v=7wjR56PRCGM
а вот сайт 
![cayspavrrxo](https://user-images.githubusercontent.com/12067451/36644540-e4148f0c-1a6c-11e8-8824-57acbfd530e3.jpg)


p.s мы ждем \@media

**Comment by AlexZetto** [comment]
Должно быть так?

![image](https://user-images.githubusercontent.com/31699385/36645035-e7d1c522-1a73-11e8-8f6f-d6d9e313190c.png)
![image](https://user-images.githubusercontent.com/31699385/36645039-f20a5194-1a73-11e8-901e-311ee064089b.png)
![image](https://user-images.githubusercontent.com/31699385/36645106-0207a398-1a75-11e8-8f9c-b7eda27f3026.png)

**Comment by djadjka** [comment]
Да, но мне бы очень хотелось чтобы десктопная и мобильные версии отличались, например в десктопная фильтра были бы сбоку

**Comment by djadjka** [comment]
Или фотографии были бы в ряд, как у тебя на мокапах

**Comment by AlexZetto** [comment]
Фильтры же сбоку
![image](https://user-images.githubusercontent.com/31699385/36645135-8e15f218-1a75-11e8-86db-6f0a6745a0cf.png)
или нужно по-другому сбоку?

**Comment by AlexZetto** [comment]
В любом случае я собираю полностью все переделывать позже.

**Comment by djadjka** [comment]
Ну типо смотри чего мы хотим. 
У тебя нет адаптивного дизайна, в тебя просто растягивается фотопосты, и все, а нам нужно чтобы твоя десктопная и мобильные версии отдичалсиь.
Посмотри видосик, я кидал в табличку

**Comment by AlexZetto** [comment]
хорошо, спасибо

**Comment by djadjka** [comment]
А смысл переделывать ? 
Не легче сделать сразу ?

**Comment by AlexZetto** [comment]
Как бы да... но всегда можно что-то изменить к лучшему.
Я хочу сделать свой проект максимально качественно.

**Comment by djadjka** [comment]
Для этого не надо думать: "ай, всеравно переделывать", обычно никчему хорошему это не приводит

**Comment by AlexZetto** [comment]
я не так думаю.

**Comment by AlexZetto** [comment]
просто мне много чего не нравится

**Comment by ala-n** [comment]
А тем временем первое исполнение задания после которого я никого не хочу убивать :)
только один момент 
figure - не совсем в том значении использован,
figure - по смыслу значит все что внутри это часть картинки, но это не так

**Comment by AlexZetto** [comment]
Его же можно доделать в течении недели?

**Comment by ala-n** [comment]
Можно.
PS: На занятии будет много полезного фи. С которым можно будет сделать perfect )

**Comment by AlexZetto** [comment]
Спасибо большое

**Comment by ala-n** [comment]
Примерчик верстки про которую спрашивала:
https://codepen.io/ala-n/pen/rJQJQe (это как один из вариантов, способов может быть много. например тут я сразу в блоки закладывал отступы, но можно пойти по другому пути и сверстать каркас не учитывая отступы. те обернуть каждый айтем еще в один блок-wrapper, и в эти блоки не вкладывать отступы (width: 50%; margin: 0;), а уже сами айтемы внутри врапера отделить отступами, например в самом врапере указать padding)
Гайды по флексбоксу:
https://html5book.ru/css3-flexbox/
https://developer.mozilla.org/ru/docs/Learn/CSS/CSS_layout/Flexbox

**Comment by AlexZetto** [comment]
спасибо :)
